### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by jtpio**
## References

Fixes https://github.com/voila-dashboards/voici/issues/4

## Code changes

- [x] Convert the docs to Markdown
- [x] Update config
- [x] Add scripts to easily build the docs locally
- [x] Add a get started guide
- [x] Mention the demo repo: https://github.com/voila-dashboards/voici-demo
- [x] Include live demo (check if `jupyterlite-sphinx` would work)

## User-facing changes

The documentation should be improved.

- Preview: https://voici--60.org.readthedocs.build/en/60/
- Voici demo app: https://voici--60.org.readthedocs.build/en/60/_static/lite/voici/tree/index.html

image

image

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
Stopping here for now. We can always iterate in follow-up PRs.

File: docs/configuration.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Configuring Voici
+
+## Templates
+
+You can different templates for your Voici application.
```
Comment (martinRenou) [inline docs/configuration.md:5]:
```suggestion
You can use different templates for your Voici application.
```

File: docs/configuration.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Templates can modify the layout and the appearance of your Voici application. Here are a few template examples:
+
+- [Voila Material](https://github.com/voila-dashboards/voila-material): Material design template for Voilà
+- [Voila Vuetify](https://github.com/voila-dashboards/voila-vuetify): Dashboard template for Voilà based on VuetifyJS
```
Comment (martinRenou) [inline docs/configuration.md:10]: Out of curiosity, have you tested that voila-vuetify works with voici?

**Review by martinRenou — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 2 above)
Comment (jtpio) [reply docs/configuration.md:10]: Partially:

image


Do you think we should remove it from the docs for now? Do we have another template that already works with Voici?

Comment (martinRenou) [reply docs/configuration.md:10]: Thanks!

Yes, let's remove it from the docs for now and figure out if we can fix its support separately. Can you open an issue for this?

The reveal and gridstack templates do work AFAIK

Comment (jtpio) [reply docs/configuration.md:10]: Yes gridstack looks good:

image

I'll update the docs to mention this one instead.

File: docs/configuration.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Configuring Voici
+
+## Templates
+
+You can use different templates for your Voici application.
+
+Templates can modify the layout and the appearance of your Voici application. Here are a few template examples:
+
+- [Voila Material](https://github.com/voila-dashboards/voila-material): Material design template for Voilà
+- [Voila Vuetify](https://github.com/voila-dashboards/voila-vuetify): Dashboard template for Voilà based on VuetifyJS
```
Comment (jtpio) [inline docs/configuration.md:10]:
```suggestion
- [Voila GridStack](https://github.com/voila-dashboards/voila-gridstack): Dashboard template for Voilà based on GridStackJS 
```

File: docs/configuration.md (hunk 2 above)
Comment (jtpio) [reply docs/configuration.md]: > Can you open an issue for this?

Probably the main issue is the compatibility with Voila 0.5.0. In this case there should already be one tracking this: https://github.com/voila-dashboards/voila-vuetify/issues/64

**Review by martinRenou — APPROVED** [review]
Thanks a lot!

**Pull request merged**